impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -362,6 +362,29 @@ public void cacheSerializingCopiers() throws Exception {
   }
 
   @Test
+  public void cacheCopierAtCacheManagerLevel() {
+    DefaultCopyProviderConfiguration copierType = new DefaultCopyProviderConfiguration()
+        .addCopierFor(Person.class, (Class)SerializingCopier.class);
```

Comment:
Well, having proper reproducible test case would make all of that clear. /cc @JSAhuja